src/udisksdaemon.c

```diff
@@ -1814,6 +1814,17 @@ udisks_daemon_get_uuid (UDisksDaemon *daemon)
 
 /* ---------------------------------------------------------------------------------------------------- */
 
+/**
+ * udisks_daemon_get_parent_for_tracking:
+ * @daemon: A #UDisksDaemon.
+ * @path: an object path of a child to find parent of
+ * @uuid_ret: a pointer to return parent UUID
+ *
+ * Finds parent block device and returns its object path and UUID.
+ * If return value is %NULL, @uuid has not been changed.
```

Comment:
`@uuid_ret` It took me a while to figure out that this function is not changing UUID of the device :-)

Reply:
Hmm, do you have any suggestion for rename/rephrase? This is still an internal daemon API.

Reply:
Just change the `@uuid` to `@uuid_ret`.

Reply:
Fixed.